landing_page_backend/contacts/utils.py

```diff
@@ -8,20 +8,16 @@
 file_name = f'contacts-{date}.csv'
 one_week_ago = date - timedelta(days=7)
 
-with open(f'contacts/reports/{file_name}', 'w') as file:
-    writer = csv.writer(file)
-    writer.writerow(
-        ('name', 'email', 'number', 'message')
-    )
-
-
-def write_data_to_csv_file():
-    contacts = Contact.objects.filter(time_create__gte=one_week_ago).values_list('name', 'email', 'number', 'message')
-
-    for contact in contacts:
-        file = open(f'contacts/reports/{file_name}', 'a')
-        with file:
-            writer = csv.writer(file)
-            writer.writerow(
-                contact
-            )
+
+def export_contacts_to_csv_file():
+    contacts = Contact.objects.filter(create__gte=one_week_ago).values_list('name', 'email', 'number', 'message')
```

Comment:
There are quite a lot of operations here, i.e. line is quite long and hard to read.
I would extract value contacts, which will just get contact.
Than I would extract value contacts_data, where I would get needed fields.